activation/activation.go

```diff
@@ -467,36 +478,57 @@ func (b *Builder) PublishActivationTx(epoch types.EpochId) error {
 	b.log.With().Info("active ids seen for epoch", log.Uint64("pos_atx_epoch", uint64(posEpoch)),
 		log.Uint32("view_cnt", activeSetSize))
 
-	signedAtx, err := b.SignAtx(atx)
-	if err != nil {
+	atxReceived := make(chan struct{})
+	b.atxPool.SetListener(func(atxHeader *types.ActivationTxHeader) {
```

Comment:
a more go approach is to pass a channel and not a delegate

Reply:
Not sure exactly what you mean.